controllers/chat/sendSignedMessage.js

```diff
@@ -63,7 +73,7 @@ const sendSignedMesage = async (req, res) => {
       });
   }
 
-  const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+  const client = new StreamChat(Environment.GETSTREAM_API_KEY, Environment.GETSTREAM_API_SECRET);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The way of initializing the StreamChat client has been changed. It now uses values from the Environment module instead of directly using process.env. This is a good practice as it centralizes the configuration management and makes the code cleaner and easier to manage.



<!-- This is an auto-generated comment by OSS CodeRabbit -->